master/networking/dual-stack.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
I understand that "dual stack" is well known term. Just wondering what people will be thinking when searching for this task. Maybe combine: Enable IPv4 and IPv6 (dual stack)?

Reply:
I like this in principle, but I don't think we can easily do this, assuming that you would also recommend changing the sidebar title to match, because of the punctuation.  I remember having a problem before with a comma in a title, and I assume I'll get the same problem with brackets.

I'll merge with this as is now, so that I can complete my sprint; but if you have a further suggestion bearing in mind the punctuation issue, please just let me know and I'll do that in a follow-up PR.